CONTRIBUTING.md

```diff
@@ -60,9 +60,13 @@ Click the icon at the right end of that box to copy that URL.
     git clone https://github.com/<UserName>/keptn.github.io
     or
     git clone git@github.com:<UserName>/keptn.github.io.git
+    or
+    gh repo clone <UserName>/keptn.github.io
     ```
     Where <*UserName*> is your github username. The keptn.github.io directory is now available in the local directory.
 
+> Be sure that [Git](https://github.com/git-guides/install-git) and [Github CLI](https://cli.github.com/) are installed on your system.
```

Comment:
Suggest moving this to right after line 40 (and replacing the colon at the end of line 40 with a period).